test/metabase/api/collection_test.clj

```diff
@@ -133,11 +133,13 @@
             (is (= (into (set (map :name (t2/select Collection))) public-collections)
                    crowbertos))
             (is (= (into public-collections #{"Crowberto Corv's Personal Collection" "Crowberto's Child Collection"})
-                   crowbertos-with-excludes))
+                   ;; TODO: better IA test data
+                   (set (remove #{"Audit" "Instance Analytics"} crowbertos-with-excludes))))
```

Comment:
I am concerned that there is a race condition, depending on if the test-resource instance analytics ran before this test or not.

I did it this way because if we remove them then it will work either way, but if we always expect them then this will have to run after the instance analytics load.